scripts/f5-config.py

```diff
@@ -14,55 +14,155 @@
 # limitations under the License.
 #
 # (c) 2014, Kevin Carter <kevin.carter@rackspace.com>
-
 import argparse
 import json
 import os
 
 import netaddr
 
+
+PART = 'RPC'
+PREFIX_NAME = 'RPC'
+
 SNAT_POOL = (
-    'create ltm snatpool %(prefix_name)s_SNATPOOL { members replace-all-with {'
-    ' %(snat_pool_addresses)s%(route_domain)s } }'
+    '### CREATE SNATPOOL ###\n'
+    'create ltm snatpool /' + PART + '/' + PREFIX_NAME + '_SNATPOOL { members replace-all-with {'
+    ' %(snat_pool_addresses)s } }'
 )
 
-MONITORS = (
-    'create ltm monitor mysql %(prefix_name)s_MON_GALERA { count 0 database'
-    ' information_schema debug yes defaults-from mysql destination *:*'
-    ' interval 30 time-until-up 0 timeout 91 username haproxy }'
-)
+#Persistance Profile:
+PERSISTANCE = [
+    r'create ltm persistence source-addr /' + PART + '/' + PREFIX_NAME + '_PROF_PERSIST_IP {'
```

Comment:
This was in the script that I started from (https://github.com/ckent99999/f5-config-generator/blob/master/f5-5-5.py). I only made the fixes listed above.

The script needs a good pep8 pass, by the way. But it would make the review even more difficult if I add that in the same patch.